src/biogeochem/CNPhenologyMod.F90

```diff
@@ -1776,36 +1789,32 @@ subroutine CropPhenology(num_pcropp, filter_pcropp                     , &
          ! from AgroIBIS subroutine planting
          ! ---------------------------------
 
-         ! in order to allow a crop to be planted only once each year
-         ! initialize cropplant = .false., but hold it = .true. through the end of the year
-
          ! initialize other variables that are calculated for crops
          ! on an annual basis in cropresidue subroutine
 
-         if ( jday == jdayyrstart(h) .and. mcsec == 0 )then
-
-            ! make sure variables aren't changed at beginning of the year
-            ! for a crop that is currently planted, such as
-            ! WINTER TEMPERATE CEREAL = winter (wheat + barley + rye)
-            ! represented here by the winter wheat pft
-
-            if (.not. croplive(p))  then
-               cropplant(p) = .false.
-               idop(p)      = NOT_Planted
-
-               ! keep next for continuous, annual winter temperate cereal crop;
-               ! if we removed elseif,
-               ! winter cereal grown continuously would amount to a cereal/fallow
-               ! rotation because cereal would only be planted every other year
-
-            else if (croplive(p) .and. (ivt(p) == nwwheat .or. ivt(p) == nirrig_wwheat)) then
-               cropplant(p) = .false.
-               !           else ! not possible to have croplive and ivt==cornORsoy? (slevis)
-            end if
```

Comment:
Does the `sowing_count` logic replace the `cropplant` logic? If so, do we need to update the winter cereal code to use the `sowing_count` logic to allow for continuous annual winter temperate cereals (as the comment implies)?

Reply:
The `sowing_count` logic is indeed intended to replace the `cropplant` logic; setting `sowing_count` equal to 0 at the beginning of the calendar year should have the same effect as this bit of code for winter wheat.

However, you and that comment have got me wondering: It _is_ possible for corn, soy, etc. to have growing seasons spanning the (hemisphere's) New Year. Since they would miss their only chance to get `cropplant` set to false, doesn't that mean they would not be planted again? E.g., for a northern-hemisphere gridcell:
- Sown Oct. 1, 2022 (`cropplant` set to true)
- Jan. 1, 2023: `croplive` true, so `cropplant` remains true
- Harvested Jan. 3, 2023; `croplive` set to false
- Sowing window 2023: Sowing block not entered because `cropplant` is still true
- Jan. 1, 2024: `croplive` false, so `cropplant` set to false

Reply:
Good questions. @samsrabin I think your latest comment implies that there might have been situations where the old code did the wrong thing but your new code would do the right thing; is that correct? I'm wondering if it would be important to do a long-ish run (e.g., 20 years or so?) comparing the new and old code to see if this new code changes answers in some situations. I'm not sure whether this is necessary or not; @samsrabin @danicalombardozzi what do you think? The purposes I would see of this are (1) understanding whether this code change fixes any bugs with the old code (intentionally or inadvertently) and if so, how often those bugs appeared; and (2) making sure that the new code isn't changing any answers in an undesirable way.

If we do this, we'll want to compare against #1628 . An easy starting point would be to run the test suite (or maybe first just a couple of tests from the test suite). But that only includes coarse-resolution crop tests (10x15 degrees) and only runs for a few years, so might not capture rare answer changes. Maybe that's enough – maybe we don't care if there are some rare situations where this branch changes answers – but I want to see what both of you think. Again, I'm happy going either way here.

Reply:
@samsrabin Thanks for pointing this out. I agree that it could have caused the original code to do the wrong thing. A test does sound like a good idea. Does running two 20-year 2-degree simulations, one with this new code and another with the original code sound reasonable? Checking the changes in crop yields would be interesting. I'm hoping that the differences aren't huge, but the test will tell!

Reply:
@billsacks I think this test is the only thing left to do before this PR is ready to go. Is there a nice automated way to perform these runs and/or compare their results for important outputs?